### PR TITLE
[v6r13] Several bug fixes

### PR DESCRIPTION
**Pull request by fstagni**
or pylint fixes

### CODE REVIEW

File: ConfigurationSystem/Client/Utilities.py [hunk 1]
```diff
@@ -486,11 +486,9 @@ def getDBParameters( fullname, defaultHost = 'localhost',
   'db' and 'queueSize'
   """
 
-  fullname = fullname
```
Comment (chaen) [inline ConfigurationSystem/Client/Utilities.py]: You need to assign the default values like it was done. So if there is no specific value, and no general value in the CS, then we take the one given in parameters
With your fixes, the values given in parameters are never used

Comment (fstagni) [inline ConfigurationSystem/Client/Utilities.py]: fullname is only used in the line below, so I don't understand your comment about this specific one.

Regarding the other default values, then there's maybe a mistake: was getOption() or getValue() to be used? If it's getValue(), then I undestand, but as the code is done the default passed ones are actually never used.

Comment (chaen) [inline ConfigurationSystem/Client/Utilities.py]: Yes, I actually meant it for the others, sorry.
But you're right, I overlooked it, this code is actually buggy and SHOULD use the one given as params if it cannot find it in the two CS places

Comment (atsareg) [inline ConfigurationSystem/Client/Utilities.py]: The logic with default values in this method is not good. It is often the case that the DB CS parameters are not defined for whatever reason ( human error, CS unavailable, etc ). This is a real bug which will prevent the DB from working. This is the only case where we allow exceptions ( at object initialization ). So, the defaults must be dropped here. I will make changes ( I have somthing else to commit as well ) and we will see

**Comment by chaen** [comment]
Sorry Andrei I don't understand what you mean, can you re explain please ?

**Comment by atsareg** [comment]
My point is that there should be no default values for the database parameters. It is more harm than convenience.

**Comment by chaen** [comment]
I disagree with this. Can you explain your reasons please ?

**Comment by atsareg** [comment]
In my long practice with installing multiple DIRAC instances it is the configuration which is the most tedious part. Faulty configuration ( or access to configuration ) can result in the databases objects failing to instantiate simply because the db access parameters are not available. If the access parameters will be taken by default in case of a failure to look them up in the CS, this will hide the real frequent problem.

**Pull request merged**